aim/web/ui/src/pages/Metrics/components/Table/SortPopover/SortPopover.tsx

```diff
@@ -78,7 +78,7 @@ function SortPopover({
     <ErrorBoundary>
       <div className='SortPopover__container'>
         <Text size={12} tint={50} className={'SortPopover__container__label'}>
-          SELECT FIELDS
+          SELECT FIELDS FOR SORTING
```

Comment:
Why we have different kind of uppercase usages? All letters are uppercase here, there are messages  with only first letters of words are uppercase, there are messages like normal sentence with only first letter with uppercase 
What is the motivation of those differences?